### PR TITLE
Removed unused translations and normalised the file

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1,153 +1,81 @@
-# encoding: UTF-8
-# Sample localization file for English. Add more files in this directory for other locales.
-# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
-
+---
 en:
-  back: Back
+  admin:
```
Comment (eoinkelly) [inline config/locales/en.yml:3]: `admin.save` is still in use in https://github.com/ODNZSL/nzsl-online/blob/staging/app/views/admin/settings/edit.html.haml#L14

**Review by eoinkelly — CHANGES_REQUESTED** [review]
LGTM other than `admin.save`

File: config/locales/en.yml (hunk 1 above)
Comment (Br3nda) [reply config/locales/en.yml:3]: i think that's missing a called to t('admin.save'). I'm a bit surprised that works

Comment (Br3nda) [reply config/locales/en.yml:3]: AHAH! went searching on apidock.com for submit_button - but it's really a helper method in this app.

i now wonder if i've missed other things translated via  `submit_button` I'll need to check more before we merge

Comment (eoinkelly) [reply config/locales/en.yml:3]: `submit_button` calls `t()` on it - see https://github.com/ODNZSL/nzsl-online/blob/staging/app/helpers/application_helper.rb#L14-L18

UI shows:

screenshot 2019-01-01 09 52 54

**Pull request closed without merging**